core/terraform/modules_move_definitions.tf

```diff
@@ -308,3 +308,8 @@ moved {
   from = module.network.azurerm_private_dns_zone.private_dns_zones["privatelink.dicom.azurehealthcareapis.com"]
   to   = azurerm_private_dns_zone.non_core["privatelink.dicom.azurehealthcareapis.com"]
 }
+
+moved {
+  from = module.network.azurerm_private_dns_zone.private_dns_zones["privatelink.azuredatabricks.net"]
+  to   = azurerm_private_dns_zone.non_core["privatelink.azuredatabricks.net"]
+}
```

Comment:
Since this will be the first version using this dns zone, I don't think we need to "move" it so this can be deleted.

Reply:
correct, removed